var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -287,6 +287,9 @@ class Gcc(AutotoolsPackage, GNUMirrorPackage):
     # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=95005
     patch('zstd.patch', when='@10')
 
+    # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=100102
+    patch('ICE_in_tsubst.patch', when='@10.3.0')
+
```

Comment:
Three comments:
1. Could we add those via URL patches so we do not have to carry the patch ourselves? Should be https://gcc.gnu.org/git/?p=gcc.git;a=patch;h=fc930b3010bd0de899a3da3209eab20664ddb703 for GCC 10 and https://gcc.gnu.org/git/?p=gcc.git;a=patch;h=f1feb74046e0feb0596b93bbb822fae02940a90e for GCC 11.
2. Why is this only applied for 10.3.0? 10.1 and 10.2 are probably also affected, right?
3. We should also patch 11.1, which is also affected according to the upstream bug.

Reply:
Thanks for the feedback.

1. Ok - sounds good.
2&3. I had only tested with 10.3.0 - but you are right the other ones should be updated as well.

I've updated the PR as suggested in a new commit.